Mathlib/SetTheory/Cardinal/Finite.lean

```diff
@@ -330,6 +330,9 @@ theorem card_eq_zero_iff_empty (α : Type*) : card α = 0 ↔ IsEmpty α := by
   rw [← Cardinal.mk_eq_zero_iff]
   simp [card]
 
+theorem card_ne_zero_iff_nonempty (α : Type*) : card α ≠ 0 ↔ Nonempty α := by
+  simp [card_eq_zero_iff_empty]
```

Comment:
Do you think this should be simp?

Reply:
Probably `ENat.card_eq_zero_iff_empty` should be, but then [Cardinal.mk_eq_zero_iff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/SetTheory/Cardinal/Defs.html#Cardinal.mk_eq_zero_iff) should also. So let's not do it in this PR :)